docsets/Carbon/docset.json

```diff
@@ -1,14 +1,28 @@
 {
     "name": "Carbon",
-    "version": "1.30.0",
-    "archive": "Carbon-1.30.0.tgz",
+    "version": "1.31.0",
+    "archive": "versions/Carbon-1.31.0.tgz",
     "author": {
-        "name": "Brian Nesbitt",
-        "link": "https://github.com/briannesbitt"
+        "name": "Adnan RIHAN",
+        "link": "https://github.com/Max13"
     },
     "aliases": [
         "PHP Carbon",
         "Carbon DateTime",
         "PHP Carbon DateTime"
+    ],
+    "specific_versions": [
+      {
+        "version": "1.31.0",
+        "archive": "versions/Carbon-1.31.0.tgz"
+      },
+      {
+        "version": "1.30.0",
+        "archive": "versions/Carbon-1.30.0.tgz"
```

Comment:
You need to actually add this archive. Moving the `txt` file doesn't do anything.